app/src/main/java/org/openobservatory/ooniprobe/model/database/Network.java

```diff
@@ -61,7 +61,7 @@ public static String getCountry(Context c, Network network) {
 	}
 
 	public static String toString(Context c, Network n) {
-		return getAsn(c, n) + " - " + getName(c, n);
+        return String.format("%s, %s ,ISP (%s)", getName(c, n), getCountry(c, n), getAsn(c, n));
```

Comment:
```suggestion
        return String.format("%s, %s (%s)", getName(c, n), getCountry(c, n), getAsn(c, n));
```

We don't need to add ",ISP" after the ISP name since it would be redundant.